automation/check-npm-version.js

```diff
@@ -7,16 +7,18 @@
  * We don't `require('semver')` to allow this script to be run as a npm
  * 'preinstall' hook, at which point no dependencies have been installed.
  */
-function semverGte(v1, v2) {
-	let v1Array, v2Array; // number[]
-	try {
-		const [, major1, minor1, patch1] = /v?(\d+)\.(\d+).(\d+)/.exec(v1);
```

Comment:
I rewrote this because typescript correctly complained that `/v?(\d+)\.(\d+).(\d+)/.exec(v1)` can return `null` which is not valid to destructure